_posts/developers/apps/inrupt-tutorial/2020-11-06-00_getting-started.md

```diff
@@ -12,92 +12,88 @@ redirect_from:
 
 # Getting Started
 
+- [Prerequisites](#prerequisites)
+- [Build the Application](#build-the-application)
+- [Run the Application](#run-the-application)
+
 This tutorial creates an introductory application that uses Inrupt's
 [JavaScript Client Libraries](https://github.com/inrupt/solid-client-js) to read
-and write your user profile data.
-
-Alternatively, to create a sample application using Inrupt's Solid React SDK,
-refer to Inrupt's [Solid React SDK documentation](https://docs.inrupt.com/developer-tools/javascript/react-sdk/).
+and write to your name to your user profile.
 
 This tutorial uses [npm](https://www.npmjs.com/get-npm) and [Parcel](https://parceljs.org/)
 to run the application locally on `localhost:1234`.
 
 ![Locally Run Getting Started Application]({{site.baseurl}}/assets/img/tutorials/locally-run-application.png)
 
+> You can also try out the application in CodeSandbox.
+>
+> [![Edit on CodeSandbox]({{site.baseUrl}}/assets/img/play-codesandbox.svg)](https://codesandbox.io/s/musing-bouman-n1ghd)
+>
+> Inside CodeSandbox, rather than running the application in the embedded browser, open the application in a new window to be able to log in.
+
 ## Prerequisites
 
-**NOTE:** This tutorial assumes the creation of a testing Pod on
-   `https://solidcommunity.net/`. If your Pod is not on 
-   `https://solidcommunity.net/` (e.g. perhaps you created your Pod on 
-   `https://pod.inrupt.com`), then you'll just need to change the value of the
-   `IDENTITY_PROVIDER` variable in the application's `index.js` file (shown
-    below) accordingly, and you can ignore the following Pod registration steps.
-   
 ### Register Your Pod and Create Your Profile
 
-If you don't already have a Pod, simply register one now:
+If you don't already have a Pod, you can register one now:
 
- - Open a browser to `https://solidcommunity.net/`.
+ 1. Open a browser to `https://solidcommunity.net/`.
 
- - Click `Register`.
+ 2. Click `Register`.
 
- - Fill in the registration form and click `Register`.
+ 3. Fill in the registration form and click `Register`. 
 
-   You'll be redirected to your public Solid Pod URL:
-   `https://<yourusername>.solidcommunity.net/`.
+    You'll be redirected to your public Solid Pod URL: `https://<yourusername>.solidcommunity.net/`.
 
 ### Install `npm`
 
 If you don't already have `npm` installed, follow the appropriate instructions
-here: [install npm](https://www.npmjs.com/get-npm). `npm` is installed as part of the
+at the [Get npm!](https://www.npmjs.com/get-npm) site. `npm` is installed as part of the
 Node.js installation.
 
 ## Build the Application
 
-1. Initialize the application
+### 1. Initialize the Application
```

Comment:
Making these sections for now.  Later, we may want to think about breaking this page up into multiple pages since its length makes it a bit unwieldy and less-than skimmable.